wings/market/coinbase_pro_market.pyx

```diff
@@ -480,6 +461,122 @@ cdef class CoinbaseProMarket(MarketBase):
                                              tracked_order.client_order_id
                                          ))
                 self.c_stop_tracking_order(tracked_order.client_order_id)
+        self._last_order_update_timestamp = current_timestamp
+
+    async def _iter_user_event_queue(self) -> AsyncIterable[Dict[str, Any]]:
+        while True:
+            try:
+                yield await self._user_stream_tracker.user_stream.get()
+            except asyncio.CancelledError:
+                raise
+            except Exception:
+                self.logger().error("Unknown error. Retrying after 1 seconds.", exc_info=True)
+                await asyncio.sleep(1.0)
+
+    async def _user_stream_event_listener(self):
+        async for event_message in self._iter_user_event_queue():
+            try:
+                content = event_message.content
+                event_type = content.get("type")
+                exchange_order_ids = [content.get("order_id"),
+                                      content.get("maker_order_id"),
+                                      content.get("taker_order_id")]
+
+                tracked_order = None
+                # There should only be one matched order, but just in case
+                for client_order_id in self._in_flight_orders:
```

Comment:
using `self._in_flight_orders.values()`will return the `InFlightOrder` itself

Reply:
fixed